testsuite/tests/warnings/deprecated_warning_specs.ml

```diff
@@ -14,6 +14,7 @@ like 'ath', is deprecated.
 Use the equivalent signed form: -f-r-a-g-i-l-e-m-a-t-h.
 Hint: Enabling or disabling a warning by its mnemonic name requires a + or - prefix.
 Hint: Did you make a spelling mistake when using a mnemonic name?
+
```

Comment:
This newline is from an empty toplevel phrase.

Reply:
Ah, I guess my move to a `signature option` should be changed back into "is the empty signature?" to account for other ways the signature could be empty. I will do this.